.github/workflows/java.yml

```diff
@@ -108,7 +108,7 @@ jobs:
               continue-on-error: true
               uses: actions/upload-artifact@v4
               with:
-                  name: test-reports-java-${{ matrix.java }}-redis-${{ matrix.redis }}-${{ matrix.host.RUNNER }}
+                  name: test-reports-java-${{ matrix.java }}-${{ matrix.engine.type }}-${{ matrix.engine.version }}-${{ matrix.host.RUNNER }}
```

Comment:
strangely, engine.type is always `valkey`

Reply:
Yeah and redis never installed. @barshaul ?